internal/zinc-core/src/main/java/xsbti/compile/AnalysisContents.java

```diff
@@ -0,0 +1,18 @@
+package xsbti.compile;
+
+/**
+ * Defines an analysis file that contains information about every incremental compile.
```

Comment:
Actually, it's about *one* incremental compile. Original docs are less than precise. TOFIX.